lib/presentation/home/widgets/home_app_bar.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+import '../../../app/di/injection_container.dart';
+import '../../../core/assets/assets.gen.dart';
+import '../../../core/localization/localization_manager.dart';
+import '../../../core/theme/palette.dart';
+import '../../../core/widgets/text/custom_text.dart';
+
+class HomeAppBar extends StatefulWidget implements PreferredSizeWidget {
+  const HomeAppBar({super.key});
+  static const String routeName = '/home-app-bar';
+
+  @override
+  State<HomeAppBar> createState() => _HomeAppBarState();
+
+  @override
+  Size get preferredSize => const Size.fromHeight(kToolbarHeight);
+}
+
+class _HomeAppBarState extends State<HomeAppBar> {
+  final _localizations = sl<LocaleCubit>().appLocalizations;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      alignment: Alignment.bottomCenter,
+      padding: const EdgeInsets.symmetric(horizontal: 16, vertical: 8),
+      child: ListTile(
+        contentPadding: EdgeInsets.zero,
+        leading: ClipRRect(
+          borderRadius: BorderRadius.circular(999.r),
+          child: Container(
+            color: Palette.green.shade400,
+            child: Assets.images.profile.image(
+              width: 48.sp,
+              height: 48.sp,
+            ),
+          ),
+        ),
```

Comment:
If this is for a circle image you can use `CircularAvatar` inside of ClipRRect which has performance issues